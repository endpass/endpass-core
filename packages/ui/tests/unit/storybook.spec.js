import registerRequireContextHook from 'babel-plugin-require-context-hook/register'; // eslint-disable-line
import initStoryshots from '@storybook/addon-storyshots';

registerRequireContextHook();
initStoryshots();
