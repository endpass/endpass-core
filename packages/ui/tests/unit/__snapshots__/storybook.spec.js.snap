// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots VButton/desktop disabled 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th />
         
        <th>
          Primary
        </th>
         
        <th>
          Secondary
        </th>
         
        <th>
          Tertiary
        </th>
         
        <th>
          Quaternary
        </th>
         
        <th>
          Ghost
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <th
          scope="row"
        >
          Normal
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-normal"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-normal"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-normal"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-normal"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-normal"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
       
      <tr>
        <th
          scope="row"
        >
          Big
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-big"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-big"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-big"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-big"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-big"
            disabled="disabled"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots VButton/desktop enabled 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th />
         
        <th>
          Primary
        </th>
         
        <th>
          Secondary
        </th>
         
        <th>
          Tertiary
        </th>
         
        <th>
          Quaternary
        </th>
         
        <th>
          Ghost
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <th
          scope="row"
        >
          Normal
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
       
      <tr>
        <th
          scope="row"
        >
          Big
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots VButton/desktop icon 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th />
         
        <th>
          Primary
        </th>
         
        <th>
          Secondary
        </th>
         
        <th>
          Tertiary
        </th>
         
        <th>
          Quaternary
        </th>
         
        <th>
          Ghost
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <th
          scope="row"
        >
          Normal
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
      </tr>
       
      <tr>
        <th
          scope="row"
        >
          Big
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-primary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-secondary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-tertiary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-quaternary size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-ghost size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label 
            </span>
             
            <i
              class="icon-atom v-button-icon icon-after theme-default"
            >
              <svg
                class="svg-atom theme-default"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-action"
                />
              </svg>
            </i>
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots VButton/desktop social 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th />
         
        <th>
          Google
        </th>
         
        <th>
          GitHub
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <th
          scope="row"
        >
          Normal
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-social size-normal"
          >
            <i
              class="icon-atom v-button-icon icon-before theme-default"
            >
              <svg
                class="svg-atom theme-default"
                style="margin-right: 5px;"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-google"
                />
              </svg>
            </i>
             
            <span
              class="v-button-label"
            >
               Google
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-social size-normal"
          >
            <i
              class="icon-atom v-button-icon icon-before theme-default"
            >
              <svg
                class="svg-atom theme-default"
                style="margin-right: 5px;"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-github"
                />
              </svg>
            </i>
             
            <span
              class="v-button-label"
            >
               GitHub
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
       
      <tr>
        <th
          scope="row"
        >
          Big
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-social size-big"
          >
            <i
              class="icon-atom v-button-icon icon-before theme-default"
            >
              <svg
                class="svg-atom theme-default"
                style="margin-right: 5px;"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-google"
                />
              </svg>
            </i>
             
            <span
              class="v-button-label"
            >
               Google
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-social size-big"
          >
            <i
              class="icon-atom v-button-icon icon-before theme-default"
            >
              <svg
                class="svg-atom theme-default"
                style="margin-right: 5px;"
                xmlns="http://www.w3.org/2000/svg"
              >
                <use
                  xlink:href="img/icons.svg#icon-github"
                />
              </svg>
            </i>
             
            <span
              class="v-button-label"
            >
               GitHub
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots VButton/desktop state 1`] = `
<div>
  <table>
    <thead>
      <tr>
        <th />
         
        <th>
          Success
        </th>
         
        <th>
          Error
        </th>
      </tr>
    </thead>
     
    <tbody>
      <tr>
        <th
          scope="row"
        >
          Normal
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-success size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-error size-normal"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
       
      <tr>
        <th
          scope="row"
        >
          Big
        </th>
         
        <td>
          <button
            class="v-button theme-default skin-success size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
         
        <td>
          <button
            class="v-button theme-default skin-error size-big"
          >
            <!---->
             
            <span
              class="v-button-label"
            >
              Button Label
            </span>
             
            <!---->
          </button>
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;

exports[`Storyshots VCheckbox/desktop array 1`] = `
<div>
  <label
    class="v-checkbox theme-default"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      name="Name"
      type="checkbox"
      value="Crypto!"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          with Crypto!
        
    </span>
  </label>
   
  <br />
  
        persons: Bob, Alice
      
</div>
`;

exports[`Storyshots VCheckbox/desktop default state 1`] = `
<div>
  <label
    class="v-checkbox theme-default"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      name="Name"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          state: false
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VCheckbox/desktop default state false 1`] = `
<div>
  <label
    class="v-checkbox theme-default"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      name="checkbox"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          state: false
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VCheckbox/desktop default state true 1`] = `
<div>
  <label
    class="v-checkbox theme-default is-checked"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      name="checkbox"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          state: true
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VCheckbox/desktop disabled 1`] = `
<div>
  <label
    class="v-checkbox theme-default is-disabled"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      disabled="disabled"
      name="checkbox"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          disabled
        
    </span>
  </label>
   
  <br />
   
  <label
    class="v-checkbox theme-default is-checked is-disabled"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      disabled="disabled"
      name="checkbox"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          disabled
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VCheckbox/desktop error 1`] = `
<div>
  <label
    class="v-checkbox theme-default is-error"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control is-danger"
      name="checkbox"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          is-error
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VCheckbox/desktop true/false attr 1`] = `
<div>
  <label
    class="v-checkbox theme-default"
  >
    <i
      class="icon-atom v-checkbox-frame theme-default"
    >
      [object Object]
    </i>
     
    <input
      class="v-checkbox-control"
      name="Name"
      type="checkbox"
      value="false"
    />
     
    <span
      class="v-checkbox-label"
    >
      
          state: no
        
    </span>
  </label>
</div>
`;

exports[`Storyshots VContentSwitcher/desktop array objects 1`] = `
<div>
  <ul
    class="v-content-switcher theme-default"
  >
    <li
      class="v-content-switcher-item is-checked"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Left section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="0"
        />
      </label>
    </li>
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Middle section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="1"
        />
      </label>
    </li>
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Middle section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="2"
        />
      </label>
    </li>
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Right section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="3"
        />
      </label>
    </li>
  </ul>
   
  <div>
    state: 0
  </div>
</div>
`;

exports[`Storyshots VContentSwitcher/desktop simple array strings 1`] = `
<div>
  <ul
    class="v-content-switcher theme-default"
  >
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Left section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="Left section"
        />
      </label>
    </li>
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Middle section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="Middle section"
        />
      </label>
    </li>
    <li
      class="v-content-switcher-item"
    >
      <label
        class="v-content-switcher-label"
      >
        
      Right section
      
        <input
          class="v-content-switcher-control"
          type="radio"
          value="Right section"
        />
      </label>
    </li>
  </ul>
   
  <div>
    state: 
  </div>
</div>
`;

exports[`Storyshots VDivider/desktop default 1`] = `
<div>
  <div
    class="v-divider theme-default"
    style="width: 378px;"
  >
    <hr
      class="v-divider-before"
    />
     
    <!---->
     
    <hr
      class="v-divider-after"
    />
  </div>
</div>
`;

exports[`Storyshots VDivider/desktop with text 1`] = `
<div>
  <div
    class="v-divider theme-default"
    style="width: 378px;"
  >
    <hr
      class="v-divider-before"
    />
     
    <span
      class="v-divider-text"
    >
      or register in with
    </span>
     
    <hr
      class="v-divider-after"
    />
  </div>
</div>
`;

exports[`Storyshots VInput/desktop disabled 1`] = `
<div>
  <div
    class="field-atom theme-default"
    style="width: 288px;"
  >
    <label
      class="label-atom theme-default is-muted"
    >
      
  Label

    </label>
     
    <div
      class="description-atom theme-default is-muted"
    >
      
  Helper text goes here

    </div>
     
    <span
      class="input-atom theme-default is-disabled"
    >
      <input
        class="input-atom-control"
        disabled="disabled"
        placeholder="Placeholder text"
      />
       
    </span>
     
    <!---->
  </div>
</div>
`;

exports[`Storyshots VInput/desktop enabled 1`] = `
<div>
  <div
    class="field-atom theme-default"
    style="width: 288px;"
  >
    <label
      class="label-atom theme-default"
    >
      
  Label

    </label>
     
    <div
      class="description-atom theme-default"
    >
      
  Helper text goes here

    </div>
     
    <span
      class="input-atom theme-default"
    >
      <input
        class="input-atom-control"
        placeholder="Placeholder text"
      />
       
    </span>
     
    <!---->
  </div>
  
        
      
</div>
`;

exports[`Storyshots VInput/desktop error 1`] = `
<div>
  <div
    class="field-atom theme-default"
    style="width: 288px;"
  >
    <label
      class="label-atom theme-default"
    >
      
  Label

    </label>
     
    <div
      class="description-atom theme-default"
    >
      
  Helper text goes here

    </div>
     
    <span
      class="input-atom theme-default is-error"
    >
      <input
        class="input-atom-control"
        placeholder="Placeholder text"
      />
       
      <i
        class="icon-atom icon-atom-error theme-default"
      >
        
      !
    
      </i>
    </span>
     
    <div
      class="error-atom theme-default"
    >
      
  Error message here

    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalCard both 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-card theme-default"
      >
        <div
          class="v-modal-card-controls"
        >
          <button
            class="v-modal-card-back"
          >
            [object Object]
          </button>
           
          <button
            class="v-modal-card-close"
          >
            [object Object]
          </button>
        </div>
         
        <header
          class="v-modal-card-header"
        >
          <h3
            class="v-modal-card-title"
          >
            Modal title
          </h3>
        </header>
         
        <div
          class="v-modal-card-body"
        >
          
            Modal body
          
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalCard closable 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-card theme-default"
      >
        <div
          class="v-modal-card-controls"
        >
          <!---->
           
          <button
            class="v-modal-card-close"
          >
            [object Object]
          </button>
        </div>
         
        <header
          class="v-modal-card-header"
        >
          <h3
            class="v-modal-card-title"
          >
            Modal title
          </h3>
        </header>
         
        <div
          class="v-modal-card-body"
        >
          
            Modal body
          
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalCard default 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-card theme-default"
      >
        <div
          class="v-modal-card-controls"
        >
          <!---->
           
          <!---->
        </div>
         
        <header
          class="v-modal-card-header"
        >
          <h3
            class="v-modal-card-title"
          >
            Modal title
          </h3>
        </header>
         
        <div
          class="v-modal-card-body"
        >
          
            Modal body
          
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalCard returnable 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-card theme-default"
      >
        <div
          class="v-modal-card-controls"
        >
          <button
            class="v-modal-card-back"
          >
            [object Object]
          </button>
           
          <!---->
        </div>
         
        <header
          class="v-modal-card-header"
        >
          <h3
            class="v-modal-card-title"
          >
            Modal title
          </h3>
        </header>
         
        <div
          class="v-modal-card-body"
        >
          
            Modal body
          
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalInfoCard both 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-info-card theme-default"
      >
        <aside
          class="v-modal-info-card-aside"
          style="background-image: url(img/info_bg.png);"
        >
          Some additional information about Endpass
        </aside>
         
        <div
          back-svg-href="img/icons.svg#icon-arrow-left"
          class="v-modal-card theme-default"
          close-svg-href="img/icons.svg#icon-close"
        >
          <div
            class="v-modal-card-controls"
          >
            <button
              class="v-modal-card-back"
            >
              [object Object]
            </button>
             
            <button
              class="v-modal-card-close"
            >
              [object Object]
            </button>
          </div>
           
          <header
            class="v-modal-card-header"
          >
            <h3
              class="v-modal-card-title"
            >
              Modal title
            </h3>
          </header>
           
          <div
            class="v-modal-card-body"
          >
              
            Modal body
          
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalInfoCard closable 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-info-card theme-default"
      >
        <aside
          class="v-modal-info-card-aside"
          style="background-image: url(img/info_bg.png);"
        >
          Some additional information about Endpass
        </aside>
         
        <div
          back-svg-href="img/icons.svg#icon-arrow-left"
          class="v-modal-card theme-default"
          close-svg-href="img/icons.svg#icon-close"
        >
          <div
            class="v-modal-card-controls"
          >
            <!---->
             
            <button
              class="v-modal-card-close"
            >
              [object Object]
            </button>
          </div>
           
          <header
            class="v-modal-card-header"
          >
            <h3
              class="v-modal-card-title"
            >
              Modal title
            </h3>
          </header>
           
          <div
            class="v-modal-card-body"
          >
              
            Modal body
          
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalInfoCard default 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-info-card theme-default"
      >
        <aside
          class="v-modal-info-card-aside"
          style="background-image: url(img/info_bg.png);"
        >
          Some additional information about Endpass
        </aside>
         
        <div
          back-svg-href="img/icons.svg#icon-arrow-left"
          class="v-modal-card theme-default"
          close-svg-href="img/icons.svg#icon-close"
        >
          <div
            class="v-modal-card-controls"
          >
            <!---->
             
            <!---->
          </div>
           
          <header
            class="v-modal-card-header"
          >
            <h3
              class="v-modal-card-title"
            >
              Modal title
            </h3>
          </header>
           
          <div
            class="v-modal-card-body"
          >
              
            Modal body
          
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots VModal/VModalInfoCard returnable 1`] = `
<div>
  <div
    class="mask-atom theme-default"
  >
    <div
      class="mask-atom-centered"
    >
      <div
        class="v-modal-info-card theme-default"
      >
        <aside
          class="v-modal-info-card-aside"
          style="background-image: url(img/info_bg.png);"
        >
          Some additional information about Endpass
        </aside>
         
        <div
          back-svg-href="img/icons.svg#icon-arrow-left"
          class="v-modal-card theme-default"
          close-svg-href="img/icons.svg#icon-close"
        >
          <div
            class="v-modal-card-controls"
          >
            <button
              class="v-modal-card-back"
            >
              [object Object]
            </button>
             
            <!---->
          </div>
           
          <header
            class="v-modal-card-header"
          >
            <h3
              class="v-modal-card-title"
            >
              Modal title
            </h3>
          </header>
           
          <div
            class="v-modal-card-body"
          >
              
            Modal body
          
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
